src/components/Checkout/LoanPrice.vue

```diff
@@ -80,25 +85,19 @@ export default {
 	computed: {
 		prices() {
 			if (this.type === 'loan') {
-				// determine how many (if any) overall additional shares are remaining
-				let remainingShares = parseFloat(this.loanAmount) - parseFloat(this.fundedAmount);
-
-				// subtract reservedAmount shares (minus our own reserved shares)
+				let remainingAmount = parseFloat(this.loanAmount) - parseFloat(this.fundedAmount);
+				// subtract reservedAmount (minus our own reserved amount)
 				// - only do this for loans that are not ending soon
 				// - for loans ending soon we just show remaining shares which are all un-reserved
 				if (!this.isExpiringSoon) {
-					remainingShares -= (parseFloat(this.reservedAmount) - parseInt(this.price, 10));
+					remainingAmount -= (parseFloat(this.reservedAmount) - parseFloat(this.price, 10));
```

Comment:
`parseFloat` doesn't take a second argument, so in this case it would just be `parseFloat(this.price)` (same comment for the other `parseFloat` usages below)